### PR TITLE
[TASK] Explain $GLOBALS['TYPO3_CONF_VARS']['BE']['defaultPermissions']

### PR DESCRIPTION
**Pull request by brotkrueml**
See also:
- https://github.com/TYPO3/typo3/blob/main/typo3/sysext/core/Classes/DataHandling/PagePermissionAssembler.php
- https://docs.typo3.org/m/typo3/reference-tsconfig/11.5/en-us/PageTsconfig/TceMain.html#permissions

Releases: main, 11.5

### CODE REVIEW

File: Documentation/Configuration/Typo3ConfVars/BE.rst [hunk 1]
```diff
@@ -719,6 +719,34 @@ $GLOBALS['TYPO3_CONF_VARS']['BE']['defaultPermissions']
    :type: array
    :Default: []
 
+   This option defines the default page permissions (`show`, `edit`, `delete`,
+   `new`, `editcontent`). The following order applies:
+
+   -  :php:`defaultPermissions` from :php:`TYPO3\CMS\Core\DataHandling\PagePermissionAssembler`
+   -  This option
```
Comment (linawolf) [inline Documentation/Configuration/Typo3ConfVars/BE.rst:726]:
```suggestion
   -  :php:`$GLOBALS['TYPO3_CONF_VARS']['BE']['defaultPermissions']` (the option described here)
```

File: Documentation/Configuration/Typo3ConfVars/BE.rst [hunk 2]
```diff
@@ -719,6 +719,34 @@ $GLOBALS['TYPO3_CONF_VARS']['BE']['defaultPermissions']
...
+   -  Page TSconfig va :ref:`TCEMAIN.permissions <t3tsconfig:pagetcemain-permissions-user-group>`
```
Comment (linawolf) [inline Documentation/Configuration/Typo3ConfVars/BE.rst:727]:
```suggestion
   -  Page TSconfig via :ref:`TCEMAIN.permissions <t3tsconfig:pagetcemain-permissions-user-group>`
```

File: Documentation/Configuration/Typo3ConfVars/BE.rst [hunk 3]
```diff
@@ -719,6 +719,34 @@ $GLOBALS['TYPO3_CONF_VARS']['BE']['defaultPermissions']
...
+
+   Example (which reflect the default permissions):
```
Comment (linawolf) [inline Documentation/Configuration/Typo3ConfVars/BE.rst:729]:
```suggestion
   Example (which reflects the default permissions):
```

File: Documentation/Configuration/Typo3ConfVars/BE.rst [hunk 4]
```diff
@@ -719,6 +719,34 @@ $GLOBALS['TYPO3_CONF_VARS']['BE']['defaultPermissions']
...
+   .. code-block:: php
+      :caption: typo3conf/AdditionalConfiguration.php
+
+      $GLOBALS['TYPO3_CONF_VARS']['BE']['defaultPermissions'] = [
+         'user' => 'show,edit,delete,new,editcontent',
+         'group' => 'show,edit,new,editcontent',
+         'everybody' => '',
+      ];
+
+   If you only deviate from the default permissions, for example for everybody,
```
Comment (linawolf) [inline Documentation/Configuration/Typo3ConfVars/BE.rst:740]: There is something odd about this sentence...

**Review by tomwarwick — APPROVED** [review]

**Pull request merged**